src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4442,6 +4442,9 @@ string Inner()
 				}
 
 				throw new Exception("Unable to convert {0} for {1} with type {2}".InvariantCultureFormat(memberValue, memberName, propertyType));
+
+				static string? splitAndJoin(string? value)
+					=> value == null ? null : string.Join(", ", Regex.Split(value, @"(\s*?,\s*|\s+)"));
```

Comment:
Am I wrong thinking that a `Regex.Replace` would be sufficient here? Also `*?` in the regex looks redundant to me :)

```suggestion
					=> value == null ? null : Regex.Replace(value, @"(\s*,\s*|\s+)", ", "));
```